monitor/host_monitoring.go

```diff
@@ -111,11 +111,7 @@ func checkHostReachability(host host.Host, settings *evergreen.Settings) error {
 	// take different action, depending on how the cloud provider reports the host's status
 	switch cloudStatus {
 	case cloud.StatusRunning:
-		// check if the host is reachable via SSH
-		reachable, err := cloudHost.IsSSHReachable()
-		if err != nil {
-			return errors.Wrapf(err, "error checking ssh reachability for host %s", host.Id)
-		}
+		reachable := true
```

Comment:
Could this entire function also be replaced by the agent asking for a task? When would this check tell us something that it doesn't?

Reply:
`GetInstanceStatus()` makes a provider-specific call. It will tell us if a host is terminated according to AWS, for example.

Reply:
Right, but if AWS terminated the host then we wouldn't be getting anything from the agent

Reply:
We use it to log that a host was terminated externally (line 146).

Reply:
sgtm